src/ipc-handlers.ts

```diff
@@ -708,11 +709,11 @@ export async function openSiteURL(
 		url.searchParams.append( 'playground-auto-login', 'true' );
 	}
 
-	shell.openExternal( url.toString() );
+	shellOpenExternalWrapper( url.toString() );
 }
 
 export async function openURL( event: IpcMainInvokeEvent, url: string ) {
-	return shell.openExternal( url );
+	return shellOpenExternalWrapper( url );
```

Comment:
I will remove this `return` and related conde on frontend on Monday, for consistency. We don't need it to be async function.